mmv1/third_party/terraform/tests/resource_sql_database_instance_test.go.erb

```diff
@@ -2956,4 +2982,19 @@ resource "google_sql_database_instance" "main" {
 		tier = "db-custom-2-13312"
 	}
 }`, instance, databaseVersion, rootPassword)
-}
\ No newline at end of file
+}
+
+func testGoogleSqlDatabaseInstance_activationPolicy(instance, databaseVersion, activationPolicy string) string {
+	return fmt.Sprintf(`
+resource "google_sql_database_instance" "instance" {
+  name                = "%s"
+  region              = "us-central1"
+  database_version    = "%s"
+  deletion_protection = false
```

Comment:
Since the field is updatable, it would be great to have deletion_protection be modifiable and set to `true` for most of the steps to ensure that the resource isn't accidentally deleted & re-created - then set it to `false` in a final step.

Reply:
Done